static/src/javascripts/projects/commercial/modules/comscore.js

```diff
@@ -0,0 +1,41 @@
+// @flow strict
+import config from 'lib/config';
+import { loadScript } from 'lib/load-script';
+import { commercialFeatures } from 'common/modules/commercial/commercial-features';
+
+type comscoreGlobals = { c1: string, c2: string, comscorekw?: string };
+
+const comscoreSrc = '//sb.scorecardresearch.com/beacon.js';
+const comscoreC1 = '2';
+const comscoreC2 = '6035250';
+
+const getGlobals = (keywords: string): comscoreGlobals => {
+    const globals: comscoreGlobals = { c1: comscoreC1, c2: comscoreC2 };
+
+    if (keywords !== 'Network Front') {
+        globals.comscorekw = keywords;
+    }
+
+    return globals;
+};
+
+export const init = (): Promise<void> => {
+    if (commercialFeatures.comscore) {
+        // eslint-disable-next-line no-underscore-dangle
+        window._comscore = window._comscore || [];
+
+        // eslint-disable-next-line no-underscore-dangle
+        window._comscore.push(getGlobals(config.get('page.keywords', '')));
+
+        loadScript(comscoreSrc, { id: 'comscore', async: true });
```

Comment:
You could consider returning `loadScript` here as `loadScript` returns a `Promise<void>`, `loadScript` rejects the `Promise` with an error if there's an error loading the script, which should mean any error will get passed on to the [catch block](https://github.com/guardian/frontend/blob/master/static/src/javascripts/bootstraps/commercial.js#L166) here and logged into Sentry.

We do this in the [init](https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js#L116) of `prepareGoogletag`.

Reply:
👍